src/status_im/core.cljs

```diff
@@ -46,6 +47,7 @@
 
 (defn init
   []
+  (status-backend-client/init)
```

Comment:
> Because if this is done, logging/setup will be called before (status-backend-client/init), resulting in problems with the call of native-module/init-status-go-logging.

Makes sense 👍🏼 

> In addition, regardless of whether config/STATUS_BACKEND_ENABLED is enabled or not, we need to call status-backend-client/init to initialize StatusBackendClient

Is this correct/ideal @qfrank? For example, why do we need to init/configure the status backend client in a release build if this client won't be used? I would expect us to check if anything should be done based on the feature flag `STATUS_BACKEND_SERVER_ENABLED` and if it's disabled it's all a no-op. Similar to how we don't setup testing stuff in a release build.

```clojure
(defn init
  []
  (when (:server-enabled? default-config)
    (set-config! default-config)))
```

Reply:
I don't want to overcomplicate things at this stage. My only goal with my comment is to help us implement in a way that doesn't allow production code to use the status backend by mistake, but I think you covered all places already pretty well.